packages/scxa-cell-type-wheel-experiment-heatmap/cypress/component/CellTypeWheelExperimentHeatmap.cy.js

```diff
@@ -5,6 +5,7 @@ import speciesResponse from '../fixtures/speciesResponse.json'
 import cellTypeWheelData from '../fixtures/CellTypeDataForWheelResponse.json'
 import heatMapTCellResponse from '../fixtures/heatMapRegulatoryTCellResponse.json'
 import URI from 'urijs'
+import {decode as base64_decode, encode as base64_encode} from 'base-64';
```

Comment:
We are not using the `decode` function in this file, so you can safely remove it.